prettier.config.js

```diff
@@ -1,17 +1,9 @@
 'use strict'
 exports.__esModule = true
 var options = {
-  trailingComma: 'none',
-  semi: false,
-  singleQuote: true,
-  proseWrap: 'always',
-  overrides: [
-    {
-      files: ['*.md', '*.mdx'],
```

Comment:
Can we keep `*.mdx` as a pattern?

Reply:
I don't fully understand your comment, can you please clarify? 

My proposal is to make the default printWidth for MD and MDX files in all Ory repos the default value of 80 chars.

If a repo needs a different printwidth, it should override the Prettier configuration locally. An example for how that looks is https://github.com/ory/docs/pull/862.

Reply:
The problem is that prettier doesn't format `mdx` per default, which is why I thought the entry is requried here. But I think this only applies to the override, and has nothing to do with what files will be formatted in a given repo.

Reply:
Prettier formats MDX files out of the box [since version 1.15](https://prettier.io/blog/2018/11/07/1.15.0.html#mdx).

One problem that might have led to your perception that it doesn't is that many repos tell Prettier exactly which files to format, e.g. `prettier --write *.md`. If we forget `*.mdx` here, Prettier won't format those files. 

Since v2.0 Prettier formats all the files it supports when calling it like this: `prettier --write .` I'm making this change in many repos (together with setting up CI to enforce formatting) separately from this PR. All of this will simplify our Prettier setup quite a bit.

Reply:
Actually, that change is already under review: https://github.com/ory/prettier-styles/pull/45/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519.

Blocking on this PR.